### DISCUSSION TITLE
sieve stopped working

### DISCUSSION THREAD

**Discussion by mholti** [docker-mailserver/docker-mailserver#3353] [Q&A] [answered]
Hi,
since one or two days DMS puts all my mails into INBOX.
I'm not aware of any changes i made.
How can I debug, why my sieve-filter isn't working anymore?
Are there any logs i can enable?

Thanks in advance
Michael

**Comment by polarathene** [comment, marked as answer]
You have not provided much information for us. Presumably you are using `:edge` with an auto-update tool like WatchTower?

A few days ago [this change was done](https://github.com/docker-mailserver/docker-mailserver/pull/3335). It was to [fix bug report from v12 release with sieve](https://github.com/docker-mailserver/docker-mailserver/issues/3296).

If v12 or v12.1 release works for you, but `:edge` fix has broken sieve, please report it with proper bug report so we have more information.

**Comment by mholti** [comment]
Hi Brennnan,
this was the right hint...
I tried (weeks ago) to edit my sieve-script with thunderbird. Thunderbird replaces .dovecot.sieve with a link to sieve/<username>.sieve
And :edge seems not to be able to follow this link. I replaced the link with the file, restarted DMS and sieve is working again.
I'll fill a bug report and close this discussion.
Thanks
Michael
